CHANGELOG.md

```diff
@@ -9,17 +9,53 @@
 - [v0.25.1](#v0251)
 - [v0.25.0](#v0250)
 
-# [v0.28.0]()
+# [v0.28.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.28.0)
 
 ## 🔦 Highlights <!-- omit in toc -->
 
 ### Smart Dialing <!-- omit in toc -->
 * When connecting to a peer we now do [happy eyeballs](https://www.rfc-editor.org/rfc/rfc8305) like dial prioritisation to prefer QUIC addresses over TCP addresses. We dial the QUIC address first and wait 250ms to dial the TCP address of the peer.
 * In our experiments we've seen little impact on latencies up to 80th percentile. 90th and 95th percentile latencies are impacted. For details see discussion on the [PR](https://github.com/libp2p/go-libp2p/pull/2260#issuecomment-1528848170).
 * For details of the address ranking logic see godoc for `swarm.DefaultDialRanker`.
-* To disable smart dialing and keep the old behaviour use the 
+* To disable smart dialing and keep the old behaviour use the
 `libp2p.NoDelayNetworkDialRanker` option.
 
+### More Metrics! <!-- omit in toc -->
+Since the last release, we've added metrics for:
+* [Holepunching](https://github.com/libp2p/go-libp2p/pull/2246)
+
+### WebTransport <!-- omit in toc -->
+* [#2251](https://github.com/libp2p/go-libp2p/pull/2251): Infer public WebTransport address from `quic-v1` addresses if both transports are using the same port for both quic-v1 and WebTransport addresses.
+* [#2271](https://github.com/libp2p/go-libp2p/pull/2271): Only add certificate hashes to WebTransport mulitaddress if listening on WebTransport
+* [#2298](https://github.com/libp2p/go-libp2p/pull/2298): Update webtransport-go to v0.5.3
+
+## Housekeeping updates <!-- omit in toc -->
+* Identify
+  * [#2303](https://github.com/libp2p/go-libp2p/pull/2303): Don't send default protocol version
+* WebSocket
+  * [#2280](https://github.com/libp2p/go-libp2p/pull/2280): Reverted back to the Gorilla library for WebSocket
+* PeerStore
+  * [#2312](https://github.com/libp2p/go-libp2p/pull/2312): Add context to PeerStore methods
+* NAT
+  * [#2248](https://github.com/libp2p/go-libp2p/pull/2248): Move NAT mapping logic out of the host
+
+## 🐞 Bugfixes <!-- omit in toc -->
+* Identify
+  * [#2325](https://github.com/libp2p/go-libp2p/pull/2325): Don't save signed peer records
+  * [#2300](https://github.com/libp2p/go-libp2p/pull/2300): Filter received addresses based on the node's remote address
```

Comment:
These two are not really bugfixes. They feed into the overarching story that we want to prevent polluting our peerstore with local addrs.